crates/optimism/trie/src/db/models/mod.rs

```diff
@@ -81,4 +81,10 @@ tables! {
         type Key = u64; // Block number
         type Value = ChangeSet;
     }
+
+    /// A mapping table from hashed addresses to their original addresses.
+    table AddressLookup {
```

Comment:
Just a non related question to this PR.
How this table will be populated for the existing addresses?

Reply:
I am working on this [here](https://github.com/op-rs/op-reth/issues/510)